core/vmpu/src/mpu_armv7m/vmpu_armv7m_mpu.c

```diff
@@ -406,6 +407,16 @@ bool vmpu_buffer_access_is_ok(int box_id, const void * addr, size_t size)
         return false;
     }
 
+    /* Check if addr range lies in page heap. */
```

Comment:
If this is going to look the same in each of the MPU drivers, can we make this a function in `page_allocator.c`, callable by the MPU drivers instead?